library/Falcon/API.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+class Falcon_API {
+	const USER_PREF_FIELD = 'falcon_preferences';
+
+	/**
+	 * Bootstrap.
+	 */
+	public static function bootstrap() {
+		register_rest_field( 'user', static::USER_PREF_FIELD, [
+			'get_callback' => [ get_called_class(), 'get_pref_field' ],
+			'update_callback' => [ get_called_class(), 'update_pref_field' ],
+			'schema' => static::get_pref_schema(),
+		] );
+	}
+
+	/**
+	 * Get the preferences field value.
+	 *
+	 * Calls out to the enabled connectors.
+	 *
+	 * @param array $data Full response data.
+	 * @return array Data for the field.
+	 */
+	public static function get_pref_field( $data ) {
```

Comment:
Why name the functions tersely like this? 
```suggestion
	public static function get_preference_field( $data ) {
```